README.md

```diff
@@ -2,10 +2,12 @@
 
 ![Build & Lint](https://github.com/schul-cloud/schulcloud-flutter/workflows/Build%20&%20Lint/badge.svg)
 
-Check the wiki for more info.
-- [Wiki](https://github.com/schul-cloud/schulcloud-flutter/wiki)
+Please check our [<kbd>wiki</kbd>][wiki] for more info:
```

Comment:
What's up with the `<kbd>` tag? Also, shouldn't links be `[tag](url)` instead of `[tag][url]`?

Reply:
`kbd`-tag: Good question. I removed it.
link: Enclosing the URL with parentheses creates an inline-style link (`[text](url)`), whereas using brackets creates a reference-style link (`[text][reference]`) whereas the reference can come later in the document — in this case the line `[wiki]: https://github.com/schul-cloud/schulcloud-flutter/wiki` at the bottom.
See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links for more details.